gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/Field.java

```diff
@@ -32,6 +33,12 @@ public abstract class Field {
 
   public abstract TypeNode type();
 
+  // If the field is annotated with google.api.field_behavior = REQUIRED, then this is true.
```

Comment:
Can we add more comments mentioning that this is used for checking if a field should be auto-populated only?